src/Symfony/Bridge/Monolog/composer.json

```
@@ -46,5 +48,11 @@
             "/Tests/"
         ]
     },
-    "minimum-stability": "dev"
+    "minimum-stability": "dev",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:joaojacome/monolog"
+        }
+    ]
```

```suggestion
    "minimum-stability": "dev"
```

possibly unintended highjack of the monoglog-bridge installation by source :)

---

ah this was intended - this fix depends on a change in Monolog. This would be removed if this MR (and the Monolog one) were to be accepted.